### ISSUE TITLE
About-us: misc styling fixes

### ISSUE THREAD

**Issue by drubgrubby** [hackforla/website#851]
### Description
Fix the styling issues from @daniellex0 's visual review of the about us page. 

### Step by step

- [x] Fix space between header text and image
- [x] More whitespace below header text
- [x] More "pink" between header and text
- [x] Make sticky nav narrower
- [x] Check sticky nav padding
- [x] Change nav donation text to "Make a donation"
- [x] Make sure the page cards are using their full 896
- [x] Expand padding inside page cards

### Resources
 hackforla.org/about-us

Danielle's figma review: 
https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1803%3A13692

**Comment by drubgrubby** [comment]
While I was at it, I fixed the wayward nav-bar issue as part of this issue.

**Comment by daniellex0** [comment]
I added my notes for some more adjustments on Figma under 'Round 3 notes' to the right (it's getting really close!):

https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=1803%3A13692

**Comment by drubgrubby** [comment]
### Progress
Against my better judgement, I decided to fix some javascript bugs (while I was there) and this has turned into a blood pressure raising...fun party. 

I have two little things to finish on this:
1) Remove the event listeners for the accordion when switching from desktop to mobile
2) Make the sticky nav stick in the right place when going from mobile to desktop.

### ETA
Saying when I think this will be done feels like I'm cursing myself.  Let me just say that if it's not done by Tuesday, look for me at the bottom of the pool with a rock tied to my foot.

**Comment by drubgrubby** [comment]
**Progress**

...okay, where was I?

The two items in the previous comment still need to be fixed.  I am going to see if @KianBadie has a few minutes to brainstorm this with me this week. 

**Blockers**
Time. (I have to get my mother's apartment cleared out this week.)

**ETA**
I'm not going to have a lot of time this week, but hopefully I can get these done by next Sunday

**Comment by pawan92** [comment]
@drubgrubby Could you please update:
Progress:
Blockers:
Availability:
ETA:

**Comment by drubgrubby** [comment]
Progress:  I did not get the JavaScript issues fixed this week. 

Blockers: I have to get my mother's apartment cleared out by this week and it's taking longer than I thought. 

Availability:  I'm hoping to get to work on Tuesday and be available through the end of the week. 

ETA: My plan is to strip the JavaScript out of this issue and make a Pull Request for all of the styling fixes by Tuesday.  I'll open a separate issue for the JavaScript bugs and finish that by the end of the week.

**Comment by drubgrubby** [comment]
As it turns out, the PR that I made for this before I was out got merged.  I had assumed that it was just closed because I was out.  My bad for not confirming that.

But also good because I was going to do this anyway.  The PR that was closed took care of all of the design things that were on this issue.  

I was also trying to do some JavaScript fixes, but I'll create a new issue for those.  

Thank you, and good night.
